routes/rendering-info/html-static.js

```diff
@@ -140,36 +157,44 @@ module.exports = {
           name: styleHashMap.default
         }
       ],
+      sophieModules: [],
       markup: svelteMarkup
-    }
+    };
 
     let isSophieVizColorDefined = false;
     let parties = renderingData.item.parties;
     if (parties !== undefined) {
       parties.forEach(party => {
         let vizPattern = /^s-viz-color-party.*/;
-        if (_.has(party, 'color.classAttribute') && vizPattern.test(party.color.classAttribute)) {
+        if (
+          _.has(party, "color.classAttribute") &&
+          vizPattern.test(party.color.classAttribute)
+        ) {
           isSophieVizColorDefined = true;
         }
-      })
+      });
     }
 
-
     if (isSophieVizColorDefined) {
-      data.stylesheets.push({
-        url: 'https://service.sophie.nzz.ch/bundle/sophie-viz-color@^1[parties].css'
+      data.sophieModules.push({
+        name: "sophie-viz-color@^1",
```

Comment:
The module name should be same as in votes and executive tool `sophie-viz-color@1` -> https://github.com/nzzdev/Q-election-votes/blob/dev/routes/rendering-info/html-static.js#L88